content/zh/docs/faq/_index.md

```diff
@@ -0,0 +1,408 @@
+  condition: mariadb.enabled
+  tags:
+    - database
+```
+
+Helm 3中， 依赖使用了同样的表达方式，现在`Chart.yaml`是这样的：
+
+```yaml
+dependencies:
+- name: mariadb
+  version: 5.x.x
+  repository: https://kubernetes-charts.storage.googleapis.com/
+  condition: mariadb.enabled
+  tags:
+    - database
+```
+
+Chart会依然下载和放置在 `charts/` 目录， 因此 `charts/` 目录中的子chart不作修改即可继续工作。
+
+### Name (或者 --generate-name) 安装时是必需的
+
+Helm 2中，如果没有提供名称， 会自动生成一个名称。在生产环境，这被证明是一个麻烦事而不是一个有用的特性。
+而在Helm 3中，如果 `helm install` 没有提供name，会抛异常。
+
+如果仍然需要一个自动生成的名称，您可以使用 `--generate-name` 创建。
+
+### 推送Chart到OCI注册中心
+
+这是一个Helm 3 中的实验性特性。使用时需要设置环境变量 `HELM_EXPERIMENTAL_OCI=1`。
+
+Chart仓库在较高层次上是一个存储和分发Chart的地址。Heml客户端打包并将Chart推送到Chart仓库中。
+简单来说，Chart仓库就是一个基本的HTTP服务器用来存放index.yaml文件和打包的chart。
+
+Chart 仓库API满足最基本的需求有一些好处，但是有些缺点开始显现出来：
+
+- Chart 仓库很难在生产环境抽象出大部分的安全性实现。在生产环境有一个认证和授权的标准API就显得格外重要。
+- Helm Chart的初始化工具用来签名和验证chart的完整性和来源，在chart的发布过程中是可选的。
+- 在多客户场景中，同一个chart可以被其他客户上传，同样的内容会被存储两次。chart仓库可以更加智能地处理
+这个问题，但并不是正式规范的一部分。
+- 在安全的多客户实现中使用单一的索引文件进行搜索、元数据信息存放和获取chart会变得困难和笨拙。
+
+Docker的分发项目（也称作Docker注册中心 v2）是Docker 注册项目的继承者。 很多主要的云供应商都提供项目
+分发，很多供应商都提供相同的产品， 这个分发项目得益于多年的强化、安全性实践和对抗测试。
+
+请查看 `helm help chart` 和 `helm help registry` 了解如何打包chart并推送到Docker注册中心的更多信息。
+
+更多信息请查看 [注册中心](https://helm.sh/docs/topics/registries/)页面。
+
+### 移除了`helm serve`
+
+`helm serve` 命令可以在你本地机器运行一个Chart仓库用于开发目的。
+然而作为一个开发工具并没有受到太多利用，并且设计上有很多问题。最终我们决定移除它，
+拆分成了一个插件。
+
+对于 `helm serve` 的类似经历，可以查看本地文件系统存储选项在 
+[ChartMuseum](https://chartmuseum.com/docs/#using-with-local-filesystem-storage) 
+和 [servecm plugin](https://github.com/jdolitsky/helm-servecm).
+
+
+### Library chart支持
+
+Helm 3 支持的一类chart称为 “library chart”。 这是一个被其他chart共享的chart，
+但是它自己不能创建发布组件。library chart的模板只能声明 `define` 元素。 全局范围
+内的非`define`内容会被简单忽略。这允许用户复用和共享可在多个chart中重复使用的代码片段。
+避免冗余和保留chart [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)。
+
+Library chart在 Chart.yaml的依赖指令中声明，安装和管理与其他chart一致。
+
+```yaml
+dependencies:
+  - name: mylib
+    version: 1.x.x
+    repository: quay.io
+```
+
+我们很高兴对开发者开放了这个特性的使用案例，以及library charts使用的最佳实践。
+
+### Chart.yaml api版本切换
+
+随着对library chart的支持以及requirements.yaml合并到Chart.yaml，客户端可以识别Helm 2的包格式而不理解
+这些新特性。因此我们将Chart.yaml的apiVersion从 `v1` 切换到了 `v2`。
+
+`helm create` 现在使用使用新格式创建chart，默认的apiVersion也切换到了这里。
+
+客户端希望同时支持两个版本，进而能够检查Chart.yaml的`apiVersion`字段去理解如何解析包格式。
+
+### XDG 基本目录支持
+
+[XDG 基本目录规范](https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html)
+是一个定义了配置、数据和缓存文件应该存储在文件系统什么位置的可移植标准。
+
+Helm 2中，Helm 将所有的信息存储在 `~/.helm` (被亲切地称为`helm home`)，可以通过设置 `$HELM_HOME` 
+环境变量修改，或者使用全局参数 `--home`。
+
+Helm 3中，Helm现在遵守XDG 基本目录规范使用以下环境变量：
+
+- `$XDG_CACHE_HOME`
+- `$XDG_CONFIG_HOME`
+- `$XDG_DATA_HOME`
+
+Helm 插件仍然使用 `$HELM_HOME` 作为 `$XDG_DATA_HOME` 的别名，以便希望将`$HELM_HOME`作为过渡环境的变量来保证向后兼容性。
+
+一些新的环境变量也通过插件环境变量来兼容以下更改：
+
+- `$HELM_PATH_CACHE` 针对缓存路径
+- `$HELM_PATH_CONFIG` 针对配置路径
+- `$HELM_PATH_DATA` 针对data路径
+
+Helm插件期望支持Heml 3 就建议使用新的环境变量。
```

Comment:
"Heml" should be "Helm".

For this line, the English is: 
> Helm plugins looking to support Helm 3 should consider using these new environment variables instead.

It means "If a Helm plugin is going to support Helm 3, the plugin author should use these new environment variables instead of the old ones".

Is this what the translation says? Thanks.

Reply:
OK, I got it. That's it.